docs/interface.rst

```diff
@@ -385,9 +385,24 @@ The follow event kinds are understood (see ``status.py`` for more details on the
 
 - ``"download-finished"``: same as upload version.
 
+- ``"invite-created"``: A new invite is created. All invite events also have a ``folder`` indicating which folder they pertain to, a ``uuid``, ``participant-name`` indicating the invitee and ``mode`` (``"read-only"`` or ``"read-write"``).
+
+- ``"invite-updated"``: An update to an invite has happened; as well as the keys from ``"invite-created"`` there may be one or all of: ``welcome``, the Welcome message from the Magic Wormhole server (you should show users the ``"motd"`` if there is one); a ``versions`` indicating the app-version support of the other side; or a ``code`` indicating the wormhole code.
```

Comment:
I wonder if this event kind should be split up into (three?) separate ones -- e.g., "invite-welcome-received", "invite-versions-received", "invite-code-received" (or similar) -- since a) there are only a few such expected "subkinds", b) separating it this way would allow consumers to hook these messages up more directly (i.e., without the extra parsing), and c) this sort of separation would be more consistent with the "terminal" messages below (where you provide separate event  kinds -- "succeeded", "failed", "rejected", "cancelled" -- for all the different ways that an invite can be completed).